packages/rich-text/package.json

```diff
@@ -41,6 +41,7 @@
 		"@wordpress/escape-html": "file:../escape-html",
 		"@wordpress/i18n": "file:../i18n",
 		"@wordpress/keycodes": "file:../keycodes",
+		"colord": "2.9.3",
```

Comment:
Side note: This is the third separate version definition of the [`colord` package](https://github.com/search?q=repo%3AWordPress%2Fgutenberg+%22colord%22+language%3AJSON&type=code&l=JSON). As a follow-up, perhaps we could try to match the version of the packages used across the project.